test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -6685,6 +6685,19 @@ BOOST_AUTO_TEST_CASE(interface_function_external)
 	CHECK_SUCCESS(text);
 }
 
+BOOST_AUTO_TEST_CASE(interface_function_default_visibility)
+{
+	char const* text = R"(
+		interface I {
+			function f();
+		}
+	)";
+	CHECK_WARNING_ALLOW_MULTI(text, (std::vector<std::string>{
+		"No visibility specified. Defaulting to \"external\".",
+		"Functions in interfaces should be declared external."
+	}));
+}
```

Comment:
Please add another test for `0.5.0`, see `interface_function_public` for how. (Sorry that I didn't comment this in the previous request.)